### DISCUSSION TITLE
Questions about the physical panel and the Statistics Panel

### DISCUSSION THREAD

**Discussion by saika8** [eProsima/Fast-DDS-monitor#206] [Q&A]
Dear developers, 
       Thank you very much for developing the powerful Fast DDS and its monitor, but I can't see any information about the physics panel and the statistics panel when I use this monitor. At the same time, I want to see the latency of fast dds when the helloworld example is running, and I don't see the line chart information, I can only see the information in the log panel and the info panel. I have followed the steps in the official documentation to configure the environment variables for fastdds_statistics, and I have installed Fast DDS Monitor 1.2.1 and Fast DDS 2.8.1 as well. 
       What can I do to fix the above problems? If there are any ideas or solutions, I would be rather grateful.
![R4(YIXTLK)V2UI9ZHMP@){A](https://github.com/eProsima/Fast-DDS-monitor/assets/87222279/228f528a-5a40-4e1e-a389-e5dae9a8da37)

**Comment by jepemi** [comment]
Hi @saika8

Please make sure that you are activating the statistics module. It is only enabled by default since Fast DDS version 2.9.0
To do this, build Fast DDS using ``-DFASTDDS_STATISTICS=ON`` at CMake configuration step.
